### DISCUSSION TITLE
About "stargazers" in the Chinese translation expresses a bad meaning

### DISCUSSION THREAD

**Discussion by AmosHuKe** [community/community#56904] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

The title in English is "stargazers", but in Chinese it is translated as "空想家".
In Chinese, "空想家" means not very good, like "a person who only imagines, but has very little power of action".
It is suggested to translate it as "标星用户", which means "user who has clicked on the star".

| 1 | 2 |  
|---|---|
| ![IMG_4731(20230601-104509)](https://github.com/community/community/assets/32262985/3db09797-0c2f-4889-81c9-bbbdc5b8b258) | ![IMG_4732(20230601-104532)](https://github.com/community/community/assets/32262985/3d71f10e-9579-4302-bee6-bd06a6674015) |

**Comment by Iron-Ham** [comment, marked as answer]
Thank you for bringing this to our attention. We'll open up an internal ticket and get this updated.
